templates/blocks/columns.twig

```diff
@@ -34,6 +34,7 @@
 									</div>
 								{% endif %}
 
+<<<<<<< HEAD
```

Comment:
Hi @pablocubico - some merge conflicts are git committed. could you please check once?

Reply:
Ugh! Probably a rebase mistake. Checking...